recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "notebook" %}
-{% set version = "6.4.11" %}
+{% set version = "6.4.12" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/notebook-{{ version }}.tar.gz
-  sha256: 709b1856a564fe53054796c80e17a67262071c86bfbdfa6b96aaa346113c555a
+  sha256: 6268c9ec9048cff7a45405c990c29ac9ca40b0bc3ec29263d218c5e01f2b4e86
 
 build:
   number: 0
```

Comment:
There seems to be a redirect in the `doc_url` to the following `URL`
https://jupyter-notebook.readthedocs.io/en/stable/